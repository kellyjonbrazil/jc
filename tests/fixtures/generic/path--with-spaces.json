{
  "path": "/User/admin/Application Support/App.app",
  "parent": "/User/admin/Application Support",
  "filename": "App.app",
  "stem": "App",
  "extension": "app",
  "path_list": [
    "/",
    "User",
    "admin",
    "Application Support",
    "App.app"
  ]
}