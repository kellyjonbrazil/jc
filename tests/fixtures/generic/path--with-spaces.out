/User/admin/Application Support/App.app